bip-newproposal.mediawiki

```diff
@@ -0,0 +1,140 @@
+<pre>
+  BIP: TBD
+  Title: PQ-Derived Schnorr: Hybrid Post-Quantum and Schnorr Signatures
```

Comment:
The title is required to be 44 characters or less (https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-header-preamble)